explorer/src/__tests__/controllers/admin/login.test.ts

```diff
@@ -0,0 +1,66 @@
+import request from 'supertest'
+import http from 'http'
+import express from 'express'
+import { Connection } from 'typeorm'
+import { closeDbConnection, getDb } from '../../../database'
+import { clearDb } from '../../testdatabase'
+import { createAdmin } from '../../../support/admin'
+import adminLogin from '../../../controllers/admin/login'
+
+const USERNAME = 'myadmin'
+const PASSWORD = 'validpassword'
+
+const app = express()
+app.use(express.json())
+app.use('/api/v1', adminLogin)
+
+let server: http.Server
+let db: Connection
+
+beforeAll(async () => {
+  db = await getDb()
+  server = app.listen(null)
+})
+afterAll(async () => {
+  if (server) {
+    server.close()
```

Comment:
is this synchronous? i thought `.close` needs a `done` callback to it.

Reply:
I pulled the example from the job run controller so also updated the other test.